### PR TITLE
Preparation before renaming repo to 'serve'

### PR DESCRIPTION
**Pull request by akochari**
## Description

The necessary changes to CI and readme file before renaming the repository to 'serve'. We are renaming the repository to be consistent with the repo naming across different components of our service. Solves issue [SS-533](https://scilifelab.atlassian.net/browse/SS-533)

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help!
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [x] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary)
- [x] I have added a reviewer for this pull request
- [x] I have added myself as an author for this pull request
- [ ] In the case I have modified settings.py, then I have also updated the studio-settings-configmap.yaml file in serve-charts

### CODE REVIEW

File: .github/workflows/e2e-tests.yaml [hunk 1]
```diff
@@ -147,4 +147,4 @@ jobs:
           ## limits ssh access and adds the ssh public key for the user which triggered the workflow
           limit-access-to-actor: true
           ## limits ssh access and adds the ssh public keys of the listed GitHub users
-          limit-access-to-users: sandstromviktor,churnikov,hamzaimran08,alfredeen,akochari
+          limit-access-to-users: churnikov,hamzaimran08,alfredeen,akochari,anondo1969
```
Comment (churnikov) [inline .github/workflows/e2e-tests.yaml:150]: Damn, good catch. Should add this to the off-boarding process if not already there.

**Review by churnikov — APPROVED** [review]
All in all LGTM, but charts need an update as well
https://github.com/search?q=repo%3AScilifelabDataCentre%2Fserve-charts%20stackn&type=code

**Comment by akochari** [comment]
> All in all LGTM, but charts need an update as well https://github.com/search?q=repo%3AScilifelabDataCentre%2Fserve-charts%20stackn&type=code

I am looking at the charts repo as well, thanks!

**Comment by alfredeen** [comment]
lgtm

**Pull request merged**